### ISSUE TITLE
radar axisLabel support removed in eCharts4

### ISSUE THREAD

**Issue by vishubm042** [apache/incubator-echarts#9282]
### One-line summary [问题简述]
please provide backward comparability for axisLabel as it was in eCharts3

### Version & Environment [版本及环境]
+ ECharts version : Echarts 4
+ Browser version [浏览器类型和版本]:chrome Version 69.0.3497.100 (Official Build) (64-bit)
+ OS Version [操作系统类型和版本]:windows 10 64bit

### Expected behaviour [期望结果]
axisLabel should show the labels on the axis of the radar chart as it did in eCharts 3.

### ECharts option [ECharts配置项]

```javascript
option = {
    title: {
        text: '基础雷达图'
    },
    tooltip: {},
    legend: {
        data: ['预算分配（Allocated Budget）', '实际开销（Actual Spending）']
    },
    radar: {
        // shape: 'circle',
        name: {
            textStyle: {
                color: '#fff',
                backgroundColor: '#999',
                borderRadius: 3,
                padding: [3, 5]
           }
        },
        axisLabel: {
            show: true
            },
        indicator: [
           { name: '销售（sales）', max: 6500},
           { name: '管理（Administration）', max: 16000},
           { name: '信息技术（Information Techology）', max: 30000},
           { name: '客服（Customer Support）', max: 38000},
           { name: '研发（Development）', max: 52000},
           { name: '市场（Marketing）', max: 25000}
        ]
    },
    series: [{
        name: '预算 vs 开销（Budget vs spending）',
        type: 'radar',
        // areaStyle: {normal: {}},
        data : [
            {
                value : [4300, 10000, 28000, 35000, 50000, 19000],
                name : '预算分配（Allocated Budget）'
            },
             {
                value : [5000, 14000, 28000, 31000, 42000, 21000],
                name : '实际开销（Actual Spending）'
            }
        ]
    }]
};

```

### Other comments [其他信息]

please add 
        axisLabel: {
            show: true
            },
in the radar option in the below demo link -
https://ecomfe.github.io/echarts-examples/public/editor.html?c=radar

**Comment by deqingli** [comment]
sorry，I don't understand what you mean, I can see the label in the demo which you show.
![image](https://user-images.githubusercontent.com/13548221/47652077-e874b800-dbbf-11e8-9ba9-bc8e6370ec47.png)

**Comment by sxmpasch** [comment]
@deqingli - I think this refers to the scale labels for each direction as seen in this example: 
http://gallery.echartsjs.com/editor.html?c=xrJG2mDna-

**Comment by 100pah** [comment]
Related issues: #8663, #8328, #8301, #8484, #8328, #8301, #8062, #8055, #8020, #7628

**Comment by vishubm042** [comment]
@deqingli  as @sxmpasch suggested previously having the option set to 
 axisLabel: {
show: true
},
would give us the option to turn on and off the highlighted area below please provide an alternative if available to achieve the same behaviour.
![radaraxislabelissue](https://user-images.githubusercontent.com/5584289/47661823-cb3fe980-dbbf-11e8-8756-9e44dca96857.JPG)

**Comment by jeoy** [comment]
Hi, @vishubm042 , check this change: https://github.com/apache/incubator-echarts/pull/10036, hope this helps

**Comment by 100pah** [comment]
Fixed in #10036
